tools/devops/Makefile

```diff
@@ -33,11 +33,15 @@ provision-xcode.csx: provision-xcode.csx.in Makefile $(TOP)/Make.config
 		-e 's#@XCODE_ROOT_PATH@#$(XCODE_DEVELOPER_ROOT)#g' \
 		$< > $@;
 
+provision-localization.csx: provision-localization.csx.in Makefile $(TOP)/Make.config
+	$(Q_GEN) sed \
+		-e 's#@WORKING_DIRECTORY@#$(PWD)#g' \
+		$< > $(TOP)/Localize/LocProject.json;
```

Comment:
No, this should have been $@ all the seds should be the same.

Reply:
Okay. How would I save this generated file in the Localize directory?